alerts/README.md

```diff
@@ -205,6 +214,14 @@ alertmanagerConfiguration:
   name: global-alertmanager-configuration
 ```
 
+## TLS Certificate Requirement
+
+In order for Prometheus installations in onboarded Greenhouse clusters to communicate with the Alertmanager component, the Ingress requires a TLS certificate for which Prometheus is configured as trusted. There are various ways in which you can generate/configure the required TLS certificate.
+
+  - You can use an automatically generated self-signed certificate by setting `alerts.auth.autoGenerateCert.enabled` to `true`. Helm will create a self-signed cert and a secret for you.
+  - You can use your own generated self-signed certificate by setting both `alerts.auth.autoGenerateCert.enabled` to `false`. You should provide the necessary values to `alerts.auth.certFile`, `alerts.auth.keyFile`, and `alerts.auth.caFile`.
```

Comment:
what do you mean with `both` here? You are just referring to `alerts.auth.autoGenerateCert.enabled` or am I missing something?

Reply:
There is no second option. I'm going to remove the word `both`
```suggestion
  - You can use your own generated self-signed certificate by setting `alerts.auth.autoGenerateCert.enabled` to `false`. You should provide the necessary values to `alerts.auth.certFile`, `alerts.auth.keyFile`, and `alerts.auth.caFile`.
```